arangod/Agency/Agent.h

```diff
@@ -435,8 +439,9 @@ class Agent : public arangodb::Thread,
                                 // waiting until _commitIndex is at end of
                                 // our log
 
+  std::atomic<unsigned long long> _leaderSince;
   /// @brief Keep track of when I last took on leadership
-  SteadyTimePoint _leaderSince;
+  //SteadyTimePoint _leaderSince;
```

Comment:
Please remove, and move comment in line 443 up in front of the variable.

Reply:
Done.